include/usgscsm/UsgsAstroSarSensorModel.h

```diff
@@ -23,9 +25,9 @@ class UsgsAstroSarSensorModel : public csm::RasterGM, virtual public csm::Settab
 
     virtual std::string getModelState() const;
 
-    static std::string constructStateFromIsd(const std::string imageSupportData, csm::WarningList *list);
+    std::string constructStateFromIsd(const std::string imageSupportData, csm::WarningList *list);
```

Comment:
These were both not static in the Framer and LineScanner, and it removed the ability to test using m_logger. Thoughts?